building/build-debs/homeworld-keysystem/src/util/fileutil/util.go

```diff
@@ -0,0 +1,57 @@
+package fileutil
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+func Exists(path string) bool {
+	f, err := os.Open(path)
+	if err == nil {
+		f.Close()
+		return true
+	} else {
+		return false
+	}
+}
+
+func EnsureIsFolder(dirname string) error {
+	fileinfo, err := os.Stat(dirname)
+	if err != nil {
+		if os.IsNotExist(err) {
+			err = os.MkdirAll(dirname, 0755)
```

Comment:
Why so permissive?

Reply:
Because this is used to create directories like /etc/homeworld/authorities/ that have contents that need to be read by other users? So we restrict access to particular private keys as necessary, but otherwise let everything be readable by world, since they aren't security-critical.